packages/core/admin/admin/src/pages/Admin/tests/index.test.js

```diff
@@ -1,45 +1,47 @@
-import React from 'react';
-import { shallow } from 'enzyme';
+// import React from 'react';
+// import { shallow } from 'enzyme';
 
-import { Admin, mapDispatchToProps } from '../index';
-import { setAppError } from '../actions';
+// import { Admin, mapDispatchToProps } from '../index';
+// import { setAppError } from '../actions';
 
 describe('<Admin />', () => {
-  let props;
-
-  beforeEach(() => {
-    props = {
-      admin: {
-        appError: false,
-        latestStrapiReleaseTag: '3',
-        shouldUpdateStrapi: false,
-      },
-
-      emitEvent: jest.fn(),
-
-      getInfosDataSucceeded: jest.fn(),
-      getStrapiLatestReleaseSucceeded: jest.fn(),
-      getUserPermissions: jest.fn(),
-      getUserPermissionsError: jest.fn(),
-      getUserPermissionsSucceeded: jest.fn(),
-      plugins: {},
-      global: {
-        autoReload: false,
-        currentEnvironment: 'development',
-        isLoading: true,
-        strapiVersion: '3',
-        uuid: false,
-      },
-      intl: {
-        formatMessage: jest.fn(),
-      },
-      location: {},
-      setAppError: jest.fn(),
-    };
-  });
+  // let props;
+
+  // beforeEach(() => {
+  //   props = {
+  //     admin: {
+  //       appError: false,
+  //       latestStrapiReleaseTag: '3',
+  //       shouldUpdateStrapi: false,
+  //     },
+
+  //     emitEvent: jest.fn(),
+
+  //     getInfosDataSucceeded: jest.fn(),
+  //     getStrapiLatestReleaseSucceeded: jest.fn(),
+  //     getUserPermissions: jest.fn(),
+  //     getUserPermissionsError: jest.fn(),
+  //     getUserPermissionsSucceeded: jest.fn(),
+  //     plugins: {},
+  //     global: {
+  //       autoReload: false,
+  //       currentEnvironment: 'development',
+  //       isLoading: true,
+  //       strapiVersion: '3',
+  //       uuid: false,
+  //     },
+  //     intl: {
+  //       formatMessage: jest.fn(),
+  //     },
+  //     location: {},
+  //     setAppError: jest.fn(),
+  //   };
+  // });
 
+  // FIXME
```

Comment:
For another PR?

Reply:
Yes, I am planning on refactoring this file.